src/main/kotlin/three/consulting/epoc/entity/Project.kt

```diff
@@ -1,22 +1,34 @@
 package three.consulting.epoc.entity
 
+import three.consulting.epoc.dto.ProjectDTO
 import java.time.LocalDate
 import java.time.LocalDateTime
 import javax.persistence.*
 
 @Table(name = "project")
 @Entity
 data class Project(
-    @ManyToOne @JoinColumn(name = "customer_id") val customerId: Customer? = null,
-    @ManyToOne @JoinColumn(name = "employee_id") val managingEmployeeId: Employee? = null,
+    @ManyToOne @JoinColumn(name = "customer_id") val customer: Customer? = null,
+    @ManyToOne @JoinColumn(name = "employee_id") val managingEmployee: Employee? = null,
```

Comment:
These fields shouldn't be nullable

Reply:
Also just above here I think the `@Table(name = "project")` annotation is redundant and can be removed. The `@Entity` already makes it a database table and the name is automatically `project` so the other annotation does nothing.